xc7/tests/iosettings/generate.py

```diff
@@ -68,10 +67,10 @@
                     ("P18", "R18"),
                 ]
         },
-    "arty":
+    "arty-full":
         {
             "BUFG":
-                "BUFGCTRL_X0Y0",
+                "BUFGCTRL_X0Y16",
```

Comment:
There should no longer be a need to explicitly specify BUFG locations now that https://github.com/SymbiFlow/symbiflow-arch-defs/pull/1237 is merged

Reply:
Ok, fixed that.